EasyProblems/Timus 1005 - Stone Pile/coin-change-dp.cpp

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
please don't make separated folder. Do something like v1 v2 in the name, and put a comment on top of the file to indicate solving method.

Reply:
ok, I thought of keeping all the codes for one problem in one place